modules/menu/static_menu.c

```diff
@@ -877,6 +878,7 @@ static int cmd_hangupall(struct re_printf *pf, void *arg)
 	}
 	else if (!pl_strcmp(&pldir, "in")) {
 		hangup_callstate(CALL_STATE_INCOMING);
+		hangup_callstate(CALL_STATE_ARRIVED);
```

Comment:
this is the drawback of adding a new state ... lots of code have to be checked and updated ...